src/utils/constants.js

```diff
@@ -27,15 +27,25 @@ export const MESSAGES = {
     successfulBarDelete: "Bar successfully deleted."
 };
 
-export const colors = {
-    teal: "#006770",
-    blue: "#002957",
-    purple: "#662261",
-    red: "#8c191b",
-    green: "#46622d",
-    orange: "#a33e1f",
-    default_button: "rgb(0, 47, 87)"
-};
+// export const colors = {
```

Comment:
I think this object is used in other places, and changing it might break things.

It would be better to put it back to how it was, then create a new object for your list...

```js
const colorNames = [
    { value: colors.teal, text: "Teal" },
    // ...
]
```

Reply:
also, the new object might be better in `EditButtonDiualog.vue`